IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -574,7 +579,24 @@ - (NSDictionary *)wipeInfo:(id<MSIDRequestContext>)context
         return nil;
     }
     
-    NSDictionary *wipeData = [NSKeyedUnarchiver unarchiveObjectWithData:(__bridge NSData *)(data)];
+    NSDictionary *wipeData;
+#if TARGET_OS_UIKITFORMAC
+    NSError *localError;
+    wipeData = [NSKeyedUnarchiver unarchivedObjectOfClass:NSDictionary.class
+                                                 fromData:(__bridge NSData *)(data)
+                                                    error:&localError];
+    if (localError)
+    {
+        if (error) *error = localError;
```

Comment:
nit: I'm not sure if we want to create a terminal error if we fail to read wipe data? It's a non crucial part of the cache.

Reply:
changed